cli/asc.js

```diff
@@ -404,6 +463,7 @@ exports.main = function main(argv, options, callback) {
   if (args.lib) {
     let lib = args.lib;
     if (typeof lib === "string") lib = lib.split(",");
+    lib = unique(lib.map(resolve));
     Array.prototype.push.apply(customLibDirs, lib.map(lib => lib.trim()));
```

Comment:
```suggestion
    customLibDirs.push(...lib.map(p => p.trim()));
```

Reply:
I guess even better touch more code and instead:
```js
    if (typeof lib === "string") lib = lib.split(",");
    lib = unique(lib.map(resolve));
    Array.prototype.push.apply(customLibDirs, lib.map(lib => lib.trim()));
```
use this:
```js
  if (typeof lib === "string") lib = lib.trim().split(/\s*,\s*/);
  lib = unique(lib.map(resolve));
  customLibDirs.push(...lib);
```

Reply:
I suppose I can just use the latter. Thanks for the suggestion.